pipeline/projects/knoedler/linkedart.py

```diff
@@ -7,100 +7,85 @@
 factory.auto_id_type = 'uuid'
 vocab.add_art_setter()
 
+from pipeline.projects.knoedler import UID_TAG_PREFIX
+
 vocab.register_aat_class("Clock", {"parent": model.HumanMadeObject, "id": "300041575", "label": "Clock"})
 vocab.register_aat_class("Cards", {"parent": model.HumanMadeObject, "id":"300211294", "label": "Playing Cards"})
-object_type_map = {
```

Comment:
I think we're going to need to improve the JSON service file loading so that we can have both project-specific files as well as things that apply across projects (like this object type map).

Reply:
:+1:

Reply:
Yep. Sounds good. I'll try to work on that once these changes get merged.